src/ResourceManager/ApiManagement/Commands.ApiManagement/Commands/NewAzureApiManagementVirtualNetwork.cs

```diff
@@ -34,25 +34,17 @@ public class NewAzureApiManagementVirtualNetwork : AzureRMCmdlet
         [Parameter(
             ValueFromPipelineByPropertyName = false,
             Mandatory = true,
-            HelpMessage = "Name of the sub network.")]
+            HelpMessage = "Subnet Resource Id.")]
         [ValidateNotNullOrEmpty]
-        public string SubnetName { get; set; }
-
-        [Parameter(
-            ValueFromPipelineByPropertyName = false,
-            Mandatory = true,
-            HelpMessage = "Identifier of the virtual network.")]
-        [ValidateNotNullOrEmpty]
-        public Guid VnetId { get; set; }
+        public string SubnetResourceId { get; set; }
 
```

Comment:
@solankisamir this is a breaking change; all existing scripts that use this cmdlet will no longer work (both `SubnetName` and `VnetId` were required parameters and are now removed). What is the reaosn behind this change?